spec/index.bs

```diff
@@ -7658,25 +7658,27 @@ dictionary GPUComputePipelineDescriptor
                     |descriptor|.{{GPUPipelineDescriptorBase/layout}} is {{GPUAutoLayoutMode/"auto"}},
                     and |descriptor|.{{GPUPipelineDescriptorBase/layout}} otherwise.
 
-                1. If any of the requirements in the following steps are unsatisfied,
-                    [$generate a validation error$], make |pipeline| [=invalid=], and stop.
+                1. All of the requirements in the following steps |must| be met.
+                    If not, [$generate a validation error$], make |pipeline| [=invalid=], and stop.
```

Comment:
I'm not really sure why, but I kinda want to expand the "If not" to something like "If any requirements are not met", but that also feels redundant. 🤷

Reply:
Ah... "if not" doesn't grammatically quite work. It should be "If they are not"

Reply:
Perfect!